packages/cbioportal-frontend-commons/src/index.tsx

```diff
@@ -60,3 +60,4 @@ export { default as TruncatedText } from './components/truncatedText/TruncatedTe
 export * from './lib/hashString';
 export * from './lib/getColor';
 export * from './components/appContext/AppContext';
+export { errorIcon, loaderIcon } from './components/StatusHelpers';
```

Comment:
Let's just export everything similar to all other exports in this file?
```suggestion
export * from './components/StatusHelpers';
```